### DISCUSSION TITLE
Trig feedback from the API for A1001?

### DISCUSSION THREAD

**Discussion by markus-ta** [AxisCommunications/discussions#7] [VAPIX] [answered]
Hello!

Do anyone know if its possible to trig the feedback test options found in the Hardware Connection Verification page for the Axis A1001?

![image](https://user-images.githubusercontent.com/60717871/173012590-e6696ca9-73ba-48b4-ade6-34b42ddceb4d.png)

**Comment by mattias-kindborg-at-work** [comment]
Hi @markus-ta! We'll look into this and hopefully return with an answer next week.

**Reply by markus-ta** [reply]
Amazing, thank you very much :)

**Reply by markus-ta** [reply]
@mattias-kindborg-at-work Any news about this :)

**Reply by mattias-kindborg-at-work** [reply]
I @markus-ta and sorry for the delay. I've been trying to find the answer to the question but with no luck. I'm going to leave the question open in case I find the answer or in case another developer knows the answer.

**Reply by markus-ta** [reply]
@mattias-kindborg-at-work Thank you for checking. wish you a nice midsummer :)

**Reply by mattias-kindborg-at-work** [reply]
Have a nice midsummer you as well!

**Comment by mattias-kindborg-at-work** [comment, marked as answer]
Wouldn't this be the *IdPoint feedback* in the *IdPoint service*? The documentation can be found in [the VAPIX Library](https://www.axis.com/vapix-library/subjects/t10067552/section/t10067528/display?section=t10067528-t10067603)?

**Reply by markus-ta** [reply]
I have completely missed that one. Just tested and it worked perfectly. Thank you very much, @mattias-kindborg-at-work !
